packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec

```diff
@@ -0,0 +1,181 @@
+BuildRequires: %{?scl_prefix}npm(babel-plugin-transform-object-rest-spread) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(babel-polyfill) >= 6.26.0
+BuildRequires: %{?scl_prefix}npm(babel-polyfill) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(babel-preset-env) >= 1.6.0
+BuildRequires: %{?scl_prefix}npm(babel-preset-env) < 2.0.0
+BuildRequires: %{?scl_prefix}npm(babel-preset-react) >= 6.5.0
+BuildRequires: %{?scl_prefix}npm(babel-preset-react) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(classnames) >= 2.2.5
+BuildRequires: %{?scl_prefix}npm(classnames) < 3.0.0
+BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
+BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(lodash) >= 4.17.11
+BuildRequires: %{?scl_prefix}npm(lodash) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(patternfly) >= 3.58.0
+BuildRequires: %{?scl_prefix}npm(patternfly) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(patternfly-react) >= 2.31.0
+BuildRequires: %{?scl_prefix}npm(patternfly-react) < 3.0.0
+BuildRequires: %{?scl_prefix}npm(prop-types) >= 15.6.2
+BuildRequires: %{?scl_prefix}npm(prop-types) < 16.0.0
+BuildRequires: %{?scl_prefix}npm(react) >= 16.8.1
+BuildRequires: %{?scl_prefix}npm(react) < 17.0.0
+BuildRequires: %{?scl_prefix}npm(react-bootstrap) >= 0.32.1
+BuildRequires: %{?scl_prefix}npm(react-bootstrap) < 1.0.0
+BuildRequires: %{?scl_prefix}npm(react-dom) >= 16.6.3
+BuildRequires: %{?scl_prefix}npm(react-dom) < 17.0.0
+BuildRequires: %{?scl_prefix}npm(react-json-tree) >= 0.11.0
+BuildRequires: %{?scl_prefix}npm(react-json-tree) < 1.0.0
+BuildRequires: %{?scl_prefix}npm(react-redux) >= 5.0.7
+BuildRequires: %{?scl_prefix}npm(react-redux) < 6.0.0
+BuildRequires: %{?scl_prefix}npm(redux) >= 3.7.2
+BuildRequires: %{?scl_prefix}npm(redux) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(redux-thunk) >= 2.3.0
+BuildRequires: %{?scl_prefix}npm(redux-thunk) < 3.0.0
+BuildRequires: %{?scl_prefix}npm(reselect) >= 3.0.1
+BuildRequires: %{?scl_prefix}npm(reselect) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(seamless-immutable) >= 7.1.3
+BuildRequires: %{?scl_prefix}npm(seamless-immutable) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(sortabular) >= 1.5.1
+BuildRequires: %{?scl_prefix}npm(sortabular) < 1.6.0
+BuildRequires: %{?scl_prefix}npm(table-resolver) >= 3.2.0
+BuildRequires: %{?scl_prefix}npm(table-resolver) < 3.3.0
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+BuildRequires: %{?scl_prefix}npm(c3) >= 0.4.11
+BuildRequires: %{?scl_prefix}npm(c3) < 1.0.0
+BuildRequires: %{?scl_prefix}npm(humanize-duration) >= 3.20.1
+BuildRequires: %{?scl_prefix}npm(humanize-duration) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(jquery) >= 2.2.4
+BuildRequires: %{?scl_prefix}npm(jquery) < 2.3.0
+BuildRequires: %{?scl_prefix}npm(react-intl) >= 2.8.0
+BuildRequires: %{?scl_prefix}npm(react-intl) < 3.0.0
+# end package.json dependencies BuildRequires
```

Comment:
Not that this blocks, but please consider moving to use of the foreman vendor JS.

Reply:
Thanks @ehelms for your remark. I had a look at other plugins and no-one is currently using it (unfortunately). 
Should I then ONLY use the foreman-vendor package and remove all other npm packages or should I add foreman-vendor and still use the npm packages?

Reply:
Plugins like `foreman_ansible` are using it. They also have additional dependencies. Probably best to ask the UI team. I'm sure @sharvit will happily guide you in the right direction.

Reply:
The build process will replace all the modules you are using with the modules that exist in the vendor anyway. (if they exist in the vendor)

It's a good practice to install the vendor as a dep and clean up modules from your package.json so your plugin declares what vendor version it needs. It will help with upgrading once there are breaking changes to the vendor since you will know about them in advance.